tests/unit/job/seedingJobCreator/SeedingJobCreator.spec.ts

```diff
@@ -281,5 +287,300 @@ describe('SeedingJobCreator', () => {
 
       expect(jobManagerClientMock.createJob).not.toHaveBeenCalled();
     });
+
+    describe('multiple seed tasks', () => {
+      it('should create multiple seed tasks when high-res parts doesnt exceed maxTilesPerSeedTask', async () => {
+        const { seedingJobCreator, queueClientMock, jobManagerClientMock, mapproxyClientMock, configMock } = seedingJobCreatorContext;
+        const baseUrl = configMock.get<string>('jobManagement.config.jobManagerBaseUrl');
+        const seedJobType = configMock.get<string>('jobManagement.ingestion.jobs.seed.type');
+        const tilesSeedingConfig = configMock.get<TilesSeedingTaskConfig>('jobManagement.ingestion.tasks.tilesSeeding');
+        const layerCacheName = 'cache-Name-s3';
+        const taskId = randomUUID();
+        const seedJobId = randomUUID();
+
+        const seedJobParams: SeedJobParams = {
+          ...seedJobParameters,
+          ingestionJob: ingestionUpdateJobHighRes,
+        };
+
+        // Get unified geometry from all parts
+        const unifiedGeometry = unifyParts(ingestionUpdateJobHighRes.parameters.partsData);
+        const seedGeometry = unifiedGeometry?.geometry;
+
+        if (!seedGeometry) {
+          throw new Error('Failed to unify parts geometry');
+        }
+
+        // Base task from 0 to zoomThreshold
+        const baseSeedTaskOptions: SeedTaskOptions = {
```

Comment:
export all those consts for an 'utils' file inside within seeding context test dir